docs/specifications/tests/RequirementsAndNormalizationOfDomainNames.md

```diff
@@ -0,0 +1,476 @@
+not contain any upper case letters.
+
+Special attention should be put on U+0049 (LATIN CAPITAL LETTER I) and U+0130
+(LATIN CAPITAL LETTER I WITH DOT ABOVE). Both must be mapped into U+0069 (LATIN
+SMALL LETTER I), also in a Turkish and Azeri locale. (See [Unicode SpecialCasing]
+for the special casing of U+0130 etc in Turkish and Azeri locale.) If a U-label
+contains a U+0131 (LATIN SMALL LETTER DOTLESS I) it will be tested as such with
+no mapping.
+
+#### A-label and U-label
+
+DNS can only handle A-labels, not U-label. In the test core suite of Zonemaster
+only A-labels are used. For normalization, all U-labels are converted to
+A-labels. Test cases will only handle an ASCII-only *Domain Name*. Conversion
+from U-label to A-label should be done as specified for [IDNA2008][RFC 5890#1.1],
+not IDNA2003.
+
+
+## Terminology
+
+No special terminology for this specification.
+
+
+[Argument list]:                         https://github.com/zonemaster/zonemaster-engine/blob/master/docs/logentry_args.md
+[B00_DOMAIN_NAME_TOO_LONG]:              #SUMMARY
+[B00_INITIAL_DOT]:                       #SUMMARY
+[B00_INVALID_ASCII]:                     #SUMMARY
+[B00_INVALID_U_LABEL]:                   #SUMMARY
+[B00_LABEL_TOO_LONG]:                    #SUMMARY
+[B00_REPEATED_DOTS]:                     #SUMMARY
+[CHARACTER TABULATION]:                  https://codepoints.net/U+0009
+[CRITICAL]:                              ../SeverityLevelDefinitions.md#critical
+[Detailed requirements]:                 #Detailed-requirements
+[EM QUAD]:                               https://codepoints.net/U+2001
+[EM SPACE]:                              https://codepoints.net/U+2003
+[EN QUAD]:                               https://codepoints.net/U+2000
+[EN SPACE]:                              https://codepoints.net/U+2002
+[ERROR]:                                 ../SeverityLevelDefinitions.md#error
+[FIGURE SPACE]:                          https://codepoints.net/U+2007
+[FOUR-PER-EM SPACE]:                     https://codepoints.net/U+2005
+[FULL STOP]:                             https://codepoints.net/U+002E
+[FULLWIDTH FULL STOP]:                   https://codepoints.net/U+FF0E
+[HAIR SPACE]:                            https://codepoints.net/U+200A
+[HALFWIDTH IDEOGRAPHIC FULL STOP]:       https://codepoints.net/U+FF61
+[HYPHEN-MINUS]:                          https://codepoints.net/U+002D
+[IDEOGRAPHIC FULL STOP]:                 https://codepoints.net/U+3002
+[IDEOGRAPHIC SPACE]:                     https://codepoints.net/U+3000
+[INFO]:                                  ../SeverityLevelDefinitions.md#info
+[LOW LINE]:                              https://codepoints.net/U+005F
+[MEDIUM MATHEMATICAL SPACE]:             https://codepoints.net/U+205F
+[NO-BREAK SPACE]:                        https://codepoints.net/U+00A0
+[NOTICE]:                                ../SeverityLevelDefinitions.md#notice
```

Comment:
I don't see where it's used, and the link should be `./SeverityLevelDefinitions.md#notice`

Reply:
Removed.